dspace/modules/xmlui/src/main/webapp/themes/Mirage/lib/css/style.css

```diff
@@ -329,6 +329,12 @@ div.ds-option-set {
     box-shadow: 6px 6px 8px rgba(33, 33, 33, 0.25);
 }
 
+.duplicate-info {
+    background-color: #c4c4ff;
```

Comment:
This color doesn't seem like an alert. Why not use the same colors as on the system alert box?  #ds-system-wide-alert, .notice.failure

Reply:
The system alert colors are difficult to notice if you're colorblind; I tested the various options using the Spectrum extension in Chrome and found that this was the highest-contrast. Honestly, we should switch all of them to something more like the purple.